### ISSUE TITLE
AGA 8 vs GPA 2172 Heating Value

### ISSUE THREAD

**Issue by morrison3873** [usnistgov/AGA8#10]
I am using the AGA 8 GERG calculated heating value provided from the spreadsheet released with the AGA 8 standard.  I am comparing this to the heating value calculated using GPA 2172.  The compressibility factor used in GPA 2172 heating value is from the AGA 8 GERG calculation.  Notice that the more complex and varying components the more the heating value varies always biased with the AGA 8 Heating value higher than the GPA 2172.

![Heating Value Difference bar chart](https://user-images.githubusercontent.com/57444781/68307811-71df8580-0071-11ea-99d7-6141693e9f88.png)
![Heating Value Comparisons](https://user-images.githubusercontent.com/57444781/68307122-5c1d9080-0070-11ea-92cc-9595eda0d794.png)
![Heating Value Comparisons less than 2000](https://user-images.githubusercontent.com/57444781/68307133-60e24480-0070-11ea-9f37-2a9eb6162378.png)

**Comment by EricLemmon** [comment]
The spreadsheet that comes with AGA8 (and that is available on this GitHub site hosted by NIST) calculates properties from three different methods:  The GROSS method from 1990, the DETAIL method from 1992, and the GERG-2008 method (from, obviously, 2008).  In the revision of AGA8 that we published in 2017, we removed all equations dealing with anything available in AGA5 or GPA 2145/2172 to avoid duplication and out-of-date values.  This included heating values.  However, one "minor" equation for the heating value remained because it is necessary to obtain the inputs to the GROSS method.  This is equation A-16 in Part 1.  This equation is a grossly (pun intended) simplified method for obtaining heating values, and should not be used other than to obtain the inputs needed in the GROSS model.  This is something we should have flagged in the xls sheet.  Your plots have shown how poorly this equation represents actual values as compared to GPA 2172.  

However, even though the equation does a poor job, it is only an intermediate step to obtain densities in the GROSS model.  It should have been labeled more as an "auxiliary equation".  With it, the calculations of densities are quite accurate within the region of validity of the equation of state.  It would be possible to make a better equation for heating values, but then the other coefficients for B and C would have to be modified to account for this change.

The new AGA8 document recommends that users implement the GERG-2008 equation when possible, followed by the use of the DETAIL method as an alternative equation, and then as a last option for obtaining properties, the GROSS method can be used.
